app/screens/modal/NoInternetConnection.tsx

```diff
@@ -52,8 +54,9 @@ const useNavigatorOnLine = () => {
 const NoInternetConnection = () => {
   const [isIgnore, setIgnore] = useState(false);
   const isOnline = useNavigatorOnLine();
+  const isClosingApp = useSelector((state: RootState) => state.ui.isClosingApp);
 
-  if (isOnline || isIgnore) {
+  if (isOnline || isIgnore || isClosingApp) {
```

Comment:
Bad solution.
Now, all modal popups should consider this `isClosingApp` state...

Let's make this `<CloseAppModal>` just on top of any others and that's it.